static/src/stylesheets/module/journalism/_politics-weekly-treat.scss

```diff
@@ -0,0 +1,190 @@
+.politics-weekly-treat {
+    font-weight: 700;
+
+    @include mq(leftCol) {
+        float: left;
+        width: $left-column;
+    }
+
+    @include mq(wide) {
+        width: $left-column-wide;
+    }
+
+    svg {
+        overflow: visible;
+    }
+}
+
+.politics-weekly-treat::after {
+    content: '';
+    width: 58px;
+    border-top: 1px solid $garnett-neutral-2;
+    margin-top: 5px;
+    display: block;
+}
+
+.politics-weekly-treat__logo svg {
+    @include mq(leftCol) {
+        width: 120px;
+        height: 120px;
+    }
+
+    @include mq(wide) {
+        width: 140px;
+        height: 140px;
+    }
+}
+
+.politics-weekly-treat__overlay {
+    @include mq(leftCol) {
+        margin-top: -35px;
+    }
+
+    @include mq(wide) {
+        margin-top: -40px;
+    }
+
+    line-height: normal;
+}
+
+.politics-weekly-treat__podcast {
+    background: $news-garnett-highlight;
+
+    color: $garnett-neutral-1;
+    font-size: 14px;
+    font-family: $f-serif-headline;
+    line-height: 1.5;
+
+    @include mq(leftCol) {
+        padding: 0 3px 3px;
+    }
+    @include mq(wide) {
+        padding: 0 3px 1px;
+    }
+
+    display: inline-block;
+}
+
+.politics-weekly-treat__podcast-title {
+    color: $garnett-neutral-5;
+    font-size: 18px;
+    font-family: $f-serif-headline;
+
+    display: inline-block;
+
+    @include mq(leftCol) {
+        line-height: 1.3;
+        padding-bottom: 5px;
+    }
+    @include mq(wide) {
+        line-height: 1.5;
+        padding-bottom: 0;
+    }
+}
+.politics-weekly-treat__podcast-title span {
+    background: $garnett-neutral-1;
+
+    @include mq(leftCol) {
+        padding: 2px 3px 5px;
+    }
+    @include mq(wide) {
+        display: inline-block;
+        padding: 1px 3px 4px;
+    }
+
+    box-decoration-break: clone;
+}
+
+.politics-weekly-treat__player {
+    background: #e9eff1;
+    font-size: 13px;
+    color: $garnett-neutral-1;
+
+    @include mq(leftCol) {
+        padding: 2px 4px 2px 1px;
+    }
+    @include mq(wide) {
+        padding: 3px 1px 4px 2px;
+    }
+
+    display: inline-flex;
+    align-items: center;
+}
+.politics-weekly-treat__player-variant-a {
+    *:first-child {
```

Comment:
Is there a way to avoid these universal selectors? Maybe we can hook into the classes of the children instead?

Reply:
The `*` can just be dropped, `:first-child` is enough. Though if you want the first _direct_ descendant, then you need to use `> :first-child`